core_api/src/modules/consultation/consultation.resolver.ts

```diff
@@ -1,6 +1,13 @@
-import { Consultation, RoleCode } from '../entities.index';
+import { Consultation, RoleCode, Patient } from '../entities.index';
 import { Resolver, Query, Arg, Authorized } from 'type-graphql';
-import { Between } from 'typeorm';
+import { Between, FindOperator } from 'typeorm';
+
+interface WhereCondition {
```

Comment:
Opinion only, for me this kind of one-use type would be better defined inline ≈ line 88 where it's used.

If not, it definitely needs a more specific name as we could imagine having several where conditions used in different queries in the same resolver file.